expectRetrying.ts

```diff
@@ -186,122 +186,119 @@ export function createExpectation(
     message,
   };
 
-  const createTextMatcher = (
+  const matchText = async (
```

Comment:
The diff here looks big because removing the inner function caused indentation to drop by one.